Build/Projects/MGCB.definition

```diff
@@ -10,6 +10,7 @@
     <Reference Include="System.Xml" />
     <Reference Include="MonoGame.Framework" />
     <Reference Include="MonoGame.Framework.Content.Pipeline" />
+    <Reference Include="MGCBReferences" />
```

Comment:
Yeah, if you make an `ExternalProject` that will let you conditionally reference another project based on the platform.